### PR TITLE
update readme documentation on manual library installation.

### PR DESCRIPTION
**Pull request by harrieshin**
### Platforms Impacted
- [x] iOS
- [x] macOS

### Description of changes

- iOS should have instructions that client needs to specify copying resource bundle.
- add separate instructions on macos.

### Verification

n/a

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1075)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,8 +41,13 @@ pod 'MicrosoftFluentUI', '~> X.X.X'
 
 - Download the latest changes from the [FluentUI for Apple](https://github.com/microsoft/fluentui-apple) repository.
 - Move the `fluentui-apple` folder into your project folder.
-- Move the relevant `FluentUI.xcodeproj` into your Xcode project depending on which platform you want to support.
-- In Xcode select your project -> your target -> General -> Embedded Binaries -> add `libFluentUI.a`.
+- For iOS platform
+    - Move the relevant `ios\FluentUI.xcodeproj` into your Xcode project depending on which platform you want to support.
```
Comment (huwilkes) [inline README.md:45]: Should this just be "Move `ios\FluentUI.xcodeproj` into your Xcode project" now? I think the relevant before was to cover both iOS and Mac .xcodeproj files. Same with Mac below.

**Review by huwilkes — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -41,8 +41,13 @@ pod 'MicrosoftFluentUI', '~> X.X.X'
 
 - Download the latest changes from the [FluentUI for Apple](https://github.com/microsoft/fluentui-apple) repository.
 - Move the `fluentui-apple` folder into your project folder.
-- Move the relevant `FluentUI.xcodeproj` into your Xcode project depending on which platform you want to support.
-- In Xcode select your project -> your target -> General -> Embedded Binaries -> add `libFluentUI.a`.
+- For iOS platform
+    - Move `ios\FluentUI.xcodeproj` into your Xcode project depending on which platform you want to support.
```
Comment (huwilkes) [inline README.md:45]: I don't think we need "depending on which platform you want to support", either.

Comment (mischreiber) [inline README.md:45]: Nit: should be a forward slash `/` in the path name.

Comment (harrieshin) [reply README.md]: wow who am i?

**Pull request merged**